src/ConnectionPool.jl

```diff
@@ -37,7 +37,6 @@ const default_connection_limit = 8
 const default_pipeline_limit = 16
 const nolimit = typemax(Int)
 
```

Comment:
Are the docs at the top of this file for the `ConnectionPool` still an accurate description of how this stuff works after these changes?